index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+            <main id="main">
+                <header class="header">
+                     <H1 id="title">Dr. Norman Borlaug</H1>
+                     <p>The man who saved a billion lives</p>
+                </header>
```

Comment:
I think it is better to use lower case h1 rather than H1 for semantic reason 👍

Reply:
You have done great to state the reason. However, I suggest to provide detailed explanation to the student to enable the student deeply understand why semantic HTML tags are important ☺️.